src/containers/Tables/OwnedDomainsTable/OwnedDomainsTableCard.tsx

```diff
@@ -55,7 +55,7 @@ const SubdomainTableCard = ({
 	const bidAmountWild = highestBid
 		? `${Number(
 				formatEther(BigNumber.from(highestBid.amount)),
-		  ).toLocaleString()} ${paymentTokenInfo.name}`
+		  ).toLocaleString()} ${paymentTokenInfo.symbol}`
```

Comment:
👍

Reply:
When I changed switched networks (to Zero) I was getting undefined for token symbol. 

<img width="899" alt="Screenshot 2022-07-14 at 23 21 31" src="https://user-images.githubusercontent.com/39112648/179098282-bc6365ea-e609-4e04-9cc2-1aa31fb46092.png">

No token symbol shown on table row 
<img width="899" alt="Screenshot 2022-07-14 at 23 22 04" src="https://user-images.githubusercontent.com/39112648/179098335-e45650d0-e781-4623-b7be-d6a4d1fca75c.png">

Reply:
@domw30  Let me know if you're still seeing the issue 1. Will be working on issue 2.